scripts/upgrade

```diff
@@ -11,13 +11,12 @@ source /usr/share/yunohost/helpers
 #=================================================
 # LOAD SETTINGS
 #=================================================
-ynh_script_progression --message="Loading installation settings..." --weight=2
+ynh_script_progression --message="Loading installation settings..." --weight=1
 
 app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
-is_public=$(ynh_app_setting_get --app=$app --key=is_public)
```

Comment:
Close! we still need to retrieve `is_public` if we want to check his value (you should add this line again, but maybe more in the `ENSURE DOWNWARD COMPATIBILITY`)

Reply:
Oh! I'm stupid, I forgot to keep this line -_-

Reply:
Don't worry, it's our job to look for mistakes...